bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx

```diff
@@ -378,10 +378,10 @@ class VideoPlayer extends Component {
   }
 
   getMuted() {
-    const { mutedByEchoTest } = this.state;
+    const { mutedByEchoTest, muted } = this.state;
     const intPlayer = this.player && this.player.getInternalPlayer();
 
-    return intPlayer && intPlayer.isMuted && intPlayer.isMuted() && !mutedByEchoTest;
+    return (intPlayer && intPlayer?.isMuted && intPlayer?.isMuted?.() && !mutedByEchoTest) || muted;
```

Comment:
I think if we check `intPlayer &&` we don't need to do `intPlayer?` in the rest of the statement...

> isMuted?.()

I've never seen optional chaining used for a function in our code before :smile_cat: I had to look it up but apparently it's a thing! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining